iocage

```diff
@@ -47,10 +47,10 @@ if [ "$(uname -K)" -lt "903000" ]; then
 fi
 
 # Check if libs are available
-if [ -e "/usr/local/lib/iocage" ] ; then
-    LIB="/usr/local/lib/iocage"
-elif [ -e "./lib" ] ; then
+if [ -e "./lib/ioc-common" ] ; then
     LIB="./lib"
+elif [ -e "/usr/local/lib/iocage" ] ; then
```

Comment:
It seemed like using `./lib` first if it happened to be the iocage lib (detect ioc-common file) would help in development, and fall back nicely to the global iocage for everyone else. If this is not desired, I can remove.